eng/pipelines/pr.yml

```diff
@@ -14,32 +14,42 @@ jobs:
 
     strategy:
       matrix:
-        # Node 18
-        "Linux - Node 18.x - Dotnet 6":
+        # Node 18 + .NET 6
+        "Linux - Node 18.x + .NET 6":
           pool: azsdk-pool-mms-ubuntu-2004-general
           imageName: ubuntu-20.04
-          nodeVersion: "18.x"
+          nodeVersion: 18.x
           dotnetVersion: 6.0.x
-
-        "Windows - Node 18.x - Dotnet 6":
+        "Windows - Node 18.x + .NET 6":
           pool: azsdk-pool-mms-win-2022-general
           imageName: windows-2022
-          nodeVersion: "18.x"
+          nodeVersion: 18.x
           dotnetVersion: 6.0.x
 
-        # Node 20
-        "Linux - Node 20.x":
+        # Node 20 + .NET 6
+        "Linux - Node 20.x + .NET 6":
           pool: azsdk-pool-mms-ubuntu-2004-general
           imageName: ubuntu-20.04
-          nodeVersion: "20.x"
+          nodeVersion: 20.x
           dotnetVersion: 6.0.x
-
-        "Windows - Node 20.x":
+        "Windows - Node 20.x + .NET 6":
           pool: azsdk-pool-mms-win-2022-general
           imageName: windows-2022
-          nodeVersion: "20.x"
+          nodeVersion: 20.x
           dotnetVersion: 6.0.x
 
+        # Node 20 + .NET 8
```

Comment:
Yeah I think here we don't need either to make a new matrix of test, we can just upgrade to dotnet 8 to build the VS extension on linux.